contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,87 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating fixes to the community
+
+The canonical source of issues in a patch release will be communicated through GitHub issues.
+
+To make it easy for the community to understand the status of each patch release, GitHub issues
+must follow a consistent format and contain basic information about the issue.
+
+GitHub issues targeted at patch releases must contain the following items:
+
+### Release label
+
+The release label contains information about which releases are impacted by the issue, and
+which release the issue is resolved in.
+
+- `release/introduced-in/1.7.0`
+- `release/resolved-in/1.7.2`
```

Comment:
Is the purpose here to be able to filter or is this just metadata?
If metadata is the important thing, we can do this like the release note block

Something like this:
If this is a cherrypick, comment out this field:
<!- -
\```
release/introduced-in=vX.Y.Z
\```
-->

Also we need a `/` command for adding `cherrypick-candidate`. Any org member should be able to do that,
but obvs the patch manager is the one that's adding the `cherrypick-approved` label.

IMO the bot could fill in `release/resolved-in/x.y.z` metadata since it can track which commits are in which releases...